ksp_plugin_adapter/burn_editor.cs

```diff
@@ -10,12 +10,10 @@ class BurnEditor : ScalingRenderer {
   public BurnEditor(PrincipiaPluginAdapter adapter,
                     Vessel vessel,
                     double initial_time,
-                    int index,
                     Func<int, BurnEditor> get_burn_at_index) {
     adapter_ = adapter;
     vessel_ = vessel;
     initial_time_ = initial_time;
-    this.index = index;
```

Comment:
Where is this set now?

Reply:
As discussed offline, every construction of a `BurnEditor` in the flight planner should be followed by a call to `UpdateAfterBurnEditorInsertionDeletion` which sets the index.  This a more systematic API than setting the index sometimes at construction and sometimes later.